+++
audience = "I"
keywords = "evolutionary algorithm,gene annotation,gene expression analysis,graph mining"
langid = "english"
isbn = "978-1-4503-7215-2"
abstract = "The identification of condition specific gene sets from transcrip- tomic experiments has important biological applications, ranging from the discovery of altered pathways between different phe- notypes to the selection of disease-related biomarkers. Statistical approaches using only gene expression data are based on an overly simplistic assumption that the genes with the most altered expres- sions are the most important in the process under study. However, a phenotype is rarely a direct consequence of the activity of a single gene, but rather reflects the interplay of several genes to perform certain molecular processes. Many methods have been proposed to analyze gene activity in the light of our knowledge about their molecular interactions. We propose, in this article, a population- based meta-heuristics based on new crossover and mutation op- erators. Our method achieves state of the art performance in an independent simulation experiment used in other studies. Applied to a public transcriptomic dataset of patients afflicted with Hepa- tocellular carcinoma, our method was able to identify significant modules of genes with meaningful biological relevance."
doi = "10.1145/3365953.3365957"
location = "New York, NY, USA, France"
publisher = "ACM"
pages = "1â€“8"
date = "2019-12-04"
authors = [ "Leandro Correa", "Denis Pallez", "Laurent Tichit", "Olivier Soriani", "Claude Pasquier",]
publication = "Proceedings of the Tenth International Conference on Computational Systems-Biology and Bioinformatics"
title = "Population-Based Meta-Heuristic for Active Modules Identification"
publication_types = [ "1",]
ID = "Correa2019"
url_pdf = "files/pdf/Correa2019.pdf"
[[links]]
name = "Article Link"
url = "https://dl.acm.org/doi/10.1145/3365953.3365957"

[[links]]
name = "Conference Site"
url = "http://csbio2019.inria.fr/"

[image]
preview_only = "true"
+++
