+++
date = "2020-04-01"
volume = "17"
title = "Transcriptome-wide-scale-predicted dsRNAs potentially involved in RNA homoeostasis are remarkably excluded from genes with no/very low expression in all developmental stages"
pages = "554â€“570"
number = "4"
keywords = "genome analysis"
publication = "RNA Biology"
issn = "1547-6286"
doi = "10.1080/15476286.2020.1717154"
authors = [ "Claude Pasquier", "Sandra Agnel", "Alain Robichon",]
impact_factor = "5.477"
audience = "I"
abstract = "RNA interference (RNAi) refers to a conserved posttranscriptional mechanism for the degradation of RNA by short dsRNAs. A genome-wide analysis of mRNAs that are complementary to RNAs of variable length that are transcribed from the full transcriptome and susceptible to being loaded onto Argonaute type 2 was performed through computational searches in the Drosophila model. We report the segments of RNAs that are complementary to mRNAs originating from introns, the exons of mRNAs and lncRNAs as a potential source of siRNAs. A full catalogue of the mRNAs that fulfill these criteria is presented, along with the quantification of multiple annealing. The catalogue was assessed for biological validation using three published lists: two for Ago2-associated RNAs and one for dsRNAs isolated from a crude extract. A broad spectrum of mRNAs were found to theoretically form intermolecular segmental dsRNAs, which should qualify them as Dicer/Ago2 substrates if they exist in vivo. These results suggest a genome-wide scale of mRNA homoeostasis via RNAi metabolism and could extend the known roles of canonical miRNAs and hairpin RNAs. The distribution of the genes for which transcripts are engaged in inter- molecular segmental pairing is largely lacking in the gene collections defined as showing no expression in each individual developmental stage from early embryos to adulthood. This trend was also observed for the genes showing very low expression from the 8-12-hour embryonic to larval stage 2. This situation was also suggested by the 3 lists generated with minimal 20-, 25- and 30-base pairing lengths."
publication_types = [ "2",]
ID = "Pasquier2020"
[[links]]
name = "Article Link"
url = "https://www.tandfonline.com/doi/full/10.1080/15476286.2020.1717154"

[[links]]
name = "Supplemental Data"
url = "supplemental/rna_biology_2020"

[[links]]
name = "Journal Site"
url = "https://www.tandfonline.com/toc/krnb20/current"

[image]
preview_only = "true"
+++
